package.json

```
@@ -354,9 +354,13 @@
             "prettier --write"
         ],
         "frontend/src/**/*.{js,jsx,mjs,ts,tsx}": [
-            "eslint --cache -c .eslintrc.js --fix",
+            "eslint --cache frontend/**/src/**/* -c .eslintrc.js --fix",
             "prettier --write"
         ],
+        "products/**/frontend/**/*.{js,jsx,mjs,ts,tsx}": [
+            "eslint --cache products/**/frontend/**/* -c .eslintrc.js --fix",
+            "prettier --write products"
```

This runs prettier on the whole `products/` directory _for each file changed_ under `products/**/frontend/**/...` – which is why this PR contains changes to a README.md, which wasn't otherwise changed.

```suggestion
            "prettier --write"
```

Do we need a separate Eslint cache BTW, if the original formatting command here didn't specify the cache path?